### DISCUSSION TITLE
Revert to classic SharePoint Search results after setting up a custom results page

### DISCUSSION THREAD

**Discussion by shazmeisteruk** [microsoft-search/pnp-modern-search#3342] [Q&A] [answered]
Hi, 
I have a strange request, i installed the latest version of PnP on the Root site of SharePoint and it is completing a tenant wide search. I showed it to my manager, showing him all the cool stuff, unfortunately he said it will confuse the users :(. Was a strange conversation! Anyway I now need to revert all the good work and change the search results page back to normal. 
I have used the following command 
**Set-PnPSearchSettings -SearchPageUrl ""**
However this is still showing after running 
**Get-PnPSearchSettings**
Redirect search URL                         : https://XXX.sharepoint.com/SitePages/Search.aspx
Which was the new search results page.  
When i run the search from the home page it is still going to the new PnP results page.
I need it to go the default M365 results page.
Any guidance would be grateful.
Thanks

**Comment by kasperbolarsen** [comment]
Hi @shazmeisteruk 
It sounds like you are looking for something like
set-pnpsearchsettings -searchscope Hub 
or 
set-pnpsearchsettings -searchscope Tenent 

depending on your setup :-)

Even though it sounds weird, coming from one of the maintainers of this project, I might have to agree with your manager. Use the out of the box Microsoft Search for default stuff and use PnP Modern Search for specific business processes / use cases where out of the box search falls short.

see https://www.techmikael.com/2020/09/microsoft-search-vs-pnp-modern-search.html

**Reply by shazmeisteruk** [reply]
Thanks for your insight Kasper , really appreciate it. I will make the suggest around using it for specific business processes in the future. 
The scope is actually fine, I had run that originally, so it is performing the tenant wide search. 
In terms of history i installed PnP and created a new search results page scoped to tenant.
Manager reviewed and said can we revert back to the original M365 search page.
This is where i am unstuck at the moment as when i am still being directed to the PnP search page i created, i cant seem to configure it back to the original search results page provided out of the box by Microsoft.

**Reply by kasperbolarsen** [reply, marked as answer]
I went into the site settings,  removed the redirection url on site and site collection level, and ran set-pnpsearchsettings -searchscope Tenent again. That reset search back to out of the box MS search

**Reply by shazmeisteruk** [reply]
Thanks Kasper - I have only removed it at Site Collection level and not Site level. It is now working.
